hw/top_earlgrey/data/ip/chip_otbn_testplan.hjson

```diff
@@ -0,0 +1,147 @@
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.ISA",
+        "OTBN.SECUREWIPE
+      ]
+    }
+    {
+      name: chip_sw_otbn_op
+      desc: '''Verify an OTBN operation.
+
+            - SW test directs the OTBN engine to perform an ECDSA operation.
+            - SW validates the reception of the otbn done interrupt once the operation is complete.
+            - SW verifies the correctness of the result with the expected value which is
+              pre-computed using a reference model.
+            '''
+      stage: V2
+      si_stage: SV2
+      tests: ["chip_sw_otbn_ecdsa_op_irq",
+              "chip_sw_otbn_ecdsa_op_irq_jitter_en"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.ISA"
+      ]
+    }
+    {
+      name: chip_sw_otbn_rnd_entropy
+      desc: '''Verify OTBN can fetch RND numbers from the entropy src.
+
+            - SW initializes the entropy subsystem to generate randomness.
+            - SW loads an OTBN app that executes instructions to read the RND bits.
+            - The OTBN app ensures that the values when read consequtively do not match, and its not
+              all 0s or all 1s, as a basic measure to ensure that the entropy subsystem is returning
+              some data.
+            '''
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_otbn_randomness"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.RND"
+      ]
+    }
+    {
+      name: chip_sw_otbn_urnd_entropy
+      desc: '''Verify OTBN can fetch URND numbers from the entropy src.
+
+            - Similar to chip_otbn_rnd_entropy, but verifies the URND bits.
+            '''
+      stage: V2
+      si_stage: SV1
+      tests:  ["chip_sw_otbn_randomness"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.RND"
+      ]
+    }
+    {
+      name: chip_sw_otbn_idle
+      desc: '''Verify the OTBN idle signal to clkmgr.
+
+            - Write the OTBN clk hint to 0 within clkmgr to indicate OTBN clk can be gated
+              and verify that the OTBN clk hint status within clkmgr reads 0 (OTBN is disabled).
+            - Write the OTBN clk hint to 1 within clkmgr to indicate OTBN clk can be enabled.
+              Verify that the OTBN clk hint status within clkmgr reads 1 (OTBN is enabled).
+            - Start an OTBN operation, write the OTBN clk hint to 0 within clkmgr and verify that
+              the OTBN clk hint status within clkmgr reads 1 (OTBN is enabled) before the
+              OTBN operation is complete.
+            - After the OTBN operation is complete, verify that the OTBN clk hint status within
+              clkmgr now reads 0 again (OTBN is disabled).
+            - Write the OTBN clk hint to 1, read and check the OTBN output for correctness.
+            '''
+      stage: V2
+      si_stage: SV2
```

Comment:
Maybe this can be bumped up to SV3? I'm not sure that much other chip bringup depends on it?

Reply:
Sure sounds reasonable.

Overall I think we will need to do a triaging pass when we're done with the planning and bump a bunch of stuff into SV3.